core/vm/gas_table.go

```diff
@@ -89,6 +89,7 @@ func memoryCopierGas(stackpos int) gasFunc {
 var (
 	gasCallDataCopy   = memoryCopierGas(2)
 	gasCodeCopy       = memoryCopierGas(2)
+	gasMemoryCopy     = memoryCopierGas(2)
```

Comment:
You could also update a comment above `memoryCopierGas` imlementation.

Reply:
https://github.com/ethereum/go-ethereum/pull/26181/commits/4d85df4844b4753efdb0a85ca22ce946c18dda40